cpp/include/cudf/round.hpp

```diff
@@ -30,23 +30,34 @@ namespace cudf {
 /**
  * @brief Different rounding methods for `cudf::round`
  */
-enum class rounding_method : int32_t { HALF_UP };
+enum class rounding_method : int32_t { HALF_UP, HALF_EVEN };
```

Comment:
Perhaps move the wikipedia links below to here instead.

Reply:
Done.